hydra/client.go

```diff
@@ -8,15 +8,51 @@ import (
 	"net/http"
 	"net/url"
 	"path"
+
+	hydrav1alpha1 "github.com/ory/hydra-maester/api/v1alpha1"
+	"github.com/ory/hydra-maester/helpers"
 )
 
-type Client struct {
+type Client interface {
```

Comment:
I think you will also need to regenerate mocks as previously they were created for this [interface](https://github.com/ory/hydra-maester/pull/83/files#diff-1c66e5dad42957f87587a8ad18d9815b912a40e6bbacb41d64be576d56a92750L46-L54)

Reply:
@piotrmsc Definitely yes. I see this is using Mockery but I'm not sure if there's supposed to be tooling to regenerate that automatically. I tried `make generate` and `go generate` without success.

Are we supposed to run this manually right now?

Reply:
I also want to point out that I'm unable to run the unit tests for that project on my development environment, for the lack of kubebuilder arm64 support (https://github.com/kubernetes-sigs/kubebuilder/issues/1932) and some other missing dependencies.

I was hoping this would run in a pipeline during the PR :)

I'll see what I can do, possibly building custom binaries on my system to make this work.



Something that probably should be tack

Reply:
I regenerated manually with mockery v1.0.0 in 3868188 and updated the tests accordingly.

Reply:
CI is running unit and integration tests on PR but does not run mockery, it's PR specific change.
Cool that you got it working! I will take a look.